frontend/src/host_orchestrator/orchestrator/cvd/cvd.go

```diff
@@ -311,10 +312,32 @@ func (cli *CLI) PowerBtn(selector Selector) error {
 }
 
 type AndroidBuild struct {
-	BuildID     string
+	BuildID string
+	// Discarded if BuildID is set.
+	// When `Branch` field is set, the latest green build id will be used.
+	Branch      string
 	BuildTarget string
 }
 
+func (b *AndroidBuild) CvdFormat() string {
+	first := b.BuildID
```

Comment:
nit: "first" renamed as "build" is more readable. And you don't need "second".

Reply:
Done.